debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -623,6 +623,16 @@ public static void j9bcutil_dumpRomClass(PrintStream out, J9ROMClassPointer romC
 			}
 		}
 
+		if (J9ROMClassHelper.hasPreloadAttribute(romClass)) {
```

Comment:
This can be put inside `if (ValueTypeHelper.getValueTypeHelper().areValueTypesSupported()) {}`

Reply:
I think that extra test would be redundant; my preference is not to add it.

Reply:
It is just to match the C implementation of `j9bcutil_dumpRomClass()` where the code to dump preload classes is inside `#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)`. But I don't have a strong preference here.

Reply:
I debated including that as well and ended up not because of its use in RomClassWalker.java https://github.com/eclipse-openj9/openj9/blob/a4b0189442e023f30d30f35cb7adc1917781d5bf/debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/RomClassWalker.java#L746-L757